extension/src/background/helpers/dataStorage.ts

```diff
@@ -34,3 +34,11 @@ export const migrateLocalStorageToBrowserStorage = async () => {
 
   await dataStorage.setItem(storage);
 };
+
+// TODO - temporary wrapper around localStorage until we replace
+// localStorage all together
+export const freighterLocalStorage = {
```

Comment:
so the idea is that when we switch over to browser.storage, we can just change what's happening inside of this function. Because of that, I think it might be better if we call this something more generic since ideally this function won't be using localStorage anymore.

Maybe you can call it `backgroundDataStorage` (or something like that)?